controller/status.go

```diff
@@ -82,3 +82,130 @@ func allStatesEqual(usl []fleet.UnitStatus) bool {
 
 	return true
 }
+
+type Status string
```

Comment:
Note that I moved this from the fleet package to here. That way the interface of `WaitForStatus` does not need to know anything about fleet statuses.